scripts/clippy

```diff
@@ -29,9 +29,12 @@ CLIPPY_LINTS=(-A clippy::module_inception  \
 	-A clippy::new_ret_no_self \
 	-A clippy::unnecessary_sort_by \
 	-A clippy::unnecessary_wraps \
+	-A clippy::bool_assert_comparison \
+	-A clippy::self_named_constructor \
+	-A clippy::enum_variant_names \
         -W clippy::dbg_macro \
         -W clippy::todo)
 
-cargo clippy --workspace --no-default-features \
+cargo clippy --workspace --fix --allow-staged --no-default-features \
```

Comment:
I believe this `--fix` breaks CI checks. In CI, it relies on non-zero exit code to check if code are formatted and linted. Now it always passes even the code breaks.